src/codeu/chat/client/commandline/Chat.java

```diff
@@ -97,6 +98,7 @@ private Panel createRootPanel(final Context context) {
     // Add a command to print a list of all commands and their description when
     // the user for "help" while on the root panel.
     //
+    // Added info command information: Priyanka Agarwal
```

Comment:
I changed the positioning of the comments and added more comments to give more information about the code I added.